app/models/repository.rb

```diff
@@ -1,3 +1,7 @@
 class Repository < KalibroClient::Entities::Processor::Repository
   include KalibroRecord
+
+  def self.latest(count=1)
+    all.sort { |one, another| another.id <=> one.id }.first(count)
```

Comment:
> And then call KalibroClient? Involving RepositoriesAttributes (which are not called currently) does not add more complexity? And more complexity on understanding what the code does? The cache is not supposed to make such queries affordable?

The added complexity is just `.map(&:repository)`. It surely beats fetching *all* repositories over the network to manually sort them. Even if we cache it, we're turning a query that should take miliseconds in one that can possibly take seconds while using up a ridiculous amount of database I/O for no reason.